README.md

```diff
@@ -156,8 +145,8 @@ Here is a brief description of what we envision for the next versions.
 | Solana SPL tokens compatibility            | Completed   |
 | Parse and resolve inline assembly          | Completed   |
 | Generate code for inline assembly          | Completed   |
-| Support Solana's Program Derived Addresses | In Progress |
-| Support latest Substrate production target | In Progress |
+| Support Solana's Program Derived Addresses | Completed   |
```

Comment:
This is my Anchor PR #1034

Reply:
#1034 should get merged in the next few days? It would be really nice to have `v0.2.0` out :blush:

Reply:
I don't think the release needs to wait for this.

Reply:
Should I put this back to `In Progress` then? Or move it to `0.3`?

Reply:
Hmm what do you think of `0.3`?

Reply:
Maybe `0.2.1`

Reply:
I did put it under `0.3`. I mean `0.3` will have this in, it is technically accurate :grin: